src/services/cycles.ts

```diff
@@ -9,6 +9,7 @@ export function getActiveCycles(dbPool: PostgresJsDatabase<typeof db>) {
       where: and(lte(db.cycles.startAt, new Date()), gte(db.cycles.endAt, new Date())),
       with: {
         questions: true,
+        options: true,
```

Comment:
with the options relationship change this will look something like: 
```
questions: {
  with: { 
    options: true
  }
}
```

Reply:
updated